src/Solvers/fourier_tridiagonal_poisson_solver.jl

```diff
@@ -39,19 +42,24 @@ function FourierTridiagonalPoissonSolver(arch, grid, planner_flag=FFTW.PATIENT)
 
     # Lower and upper diagonals are the same
     CUDA.@allowscalar begin
-        ld = arch_array(arch, [1/ΔzF(1, 1, k, grid) for k in 1:Nz-1])
-        ud = ld
+        lower_diagonal = arch_array(arch, [1 / ΔzC(1, 1, k, grid) for k in 2:Nz])
+        upper_diagonal = lower_diagonal
     end
 
     # Compute diagonal coefficients for each grid point
-    D = arch_array(arch, zeros(Nx, Ny, Nz))
-    event = launch!(arch, grid, :xy, compute_diagonals!, D, grid, λx, λy,
+    diagonal = arch_array(arch, zeros(Nx, Ny, Nz))
+    event = launch!(arch, grid, :xy, compute_diagonals!, diagonal, grid, λx, λy,
                     dependencies=Event(device(arch)))
     wait(device(arch), event)
 
     # Set up batched tridiagonal solver
     rhs_storage = arch_array(arch, zeros(complex(eltype(grid)), size(grid)...))
-    btsolver = BatchedTridiagonalSolver(arch, dl=ld, d=D, du=ud, f=rhs_storage, grid=grid)
+    btsolver = BatchedTridiagonalSolver(arch,
+                                        dl = lower_diagonal,
+                                         d = diagonal,
+                                        du = upper_diagonal,
+                                         f = rhs_storage,
+                                        grid = grid)
```

Comment:
Does this need to be aligned with the lines above?

Reply:
Good catch!